meson.build

```diff
@@ -129,170 +129,200 @@ thread_dep = dependency('threads') # pthread for tensorflow-lite
 # Protobuf
 protobuf_dep = dependency('protobuf', version: '>= 3.6.1', required: false)
 
-# Orc
-have_orcc = false
-
-if get_option('enable-orc')
-  orc_dep = dependency('orc-0.4', version: '>= 0.4.17', required: true)
-  orcc = find_program('orcc', required: true)
-  if orc_dep.found() and orcc.found()
-    have_orcc = true
-    orcc_args = [orcc, '--include', 'glib.h']
-    add_project_arguments('-DHAVE_ORC=1', language: ['c', 'cpp'])
-  else
-    error('Cannot find orc library')
-  endif
-else
-  add_project_arguments('-DDISABLE_ORC=1', language: ['c', 'cpp'])
-endif
+json_glib_dep = dependency('json-glib-1.0', required: get_option('nnfw-runtime-support')) 
 
-# NO Video support
-if get_option('disable-video-support')
-  add_project_arguments('-DNO_VIDEO=1', language: ['c', 'cpp'])
-  message('Disable Video Type Support')
-endif
+#orc
+pg_orcc = find_program('orcc', required: get_option('orcc-support'))
+orc_dep = dependency('orc-0.4', version: '>= 0.4.17', required: get_option('orcc-support'))
 
-# NO Audio support
-if get_option('disable-audio-support')
-  add_project_arguments('-DNO_AUDIO=1', language: ['c', 'cpp'])
-  message('Disable Audio Type Support')
+#movidius
+mvncsdk2_dep = cc.find_library('mvnc', required: get_option('mvncsdk2-support'))
```

Comment:
https://mesonbuild.com/Reference-manual.html
> required By default, required is set to true and Meson will abort if no program can be found. If required is set to false, Meson continue even if none of the programs can be found. You can then use the .found() method on the returned object to check whether it was found or not. Since 0.47.0 the value of a feature option can also be passed to the required keyword argument.

if mvncsdk2-support=true, meson build fails if we only have corresponding headers. (LINE 141-143 is never executed)

Reply:
Thanks. Fixed :)